docs/testing/testplan.md

```diff
@@ -0,0 +1,427 @@
+   c. Once your wallet provider completes all sign-up and verification processes, you should be able to:
+   - Setup and manage your wallet accounts, currencies, balances etc.
+   - Obtain the wallet address or payment pointer for your digital wallet.
+
+### Test Playground
+
+Use the [Web Monetization playground](https://webmonetization.org/play/) to quickly create the conditions of a monetized website in a playground area, to use only for testing purposes.
+
+When you add a wallet address or payment pointer to the playground, it is added as a receiving wallet on the playground.  
+So, the playground behaves like a monetized website on which you can use the Web Monetization extension to pay.
+
+On the playground, you can add a wallet that uses real money from a real wallet, or you can add a wallet that uses “play” money from a “test wallet”.
+
+**Important**
+
+1. The playground itself is not web monetized. We do not receive any payments from you if you use this site.
+2. The playground becomes monetized when you add one or more receiving wallet addresses to it.
+3. If your Web Monetization extension is connected to a real wallet, then your extension will facilitate payments from your real wallet (i.e. real money).
+4. Similarly, if your extension is connected to a play wallet (e.g. rafiki.money), then the extension facilitates payments using play money.
+5. If you add your own wallet address to the playground, and then use the extension to pay the playground, you are paying yourself.
+
+### Test Data
+
+This section lists websites and wallets that you can pay using the Web Monetization extension.
+
+#### Use a test wallet & play money
+
+If you have a [test wallet][testWallet] address connected to the extension, then you can use the Web Monetization extension to pay with "play" money. You have two options:
+
+1. Visit any of these monetized websites that use a [rafiki.money](https://rafiki.money) test wallet:
+
+   - https://ahimsakids.com
+   - https://alexlakatos.com
+   - https://radu.sh
+   - https://radu.sh/iframe (if you'd like to test how it works with iframes)
+
+2. Visit the Web Monetization Playground, and add test payment pointers. Here are few examples that you can use:
+   - https://ilp.rafiki.money/interledger-wm
+   - https://ilp.rafiki.money/wm-dev
+   - https://ilp.rafiki.money/laka_web
+   - https://ilp.rafiki.money/wm-receiving
+
+#### Websites and wallets using Fynbos US
+
+If you want to test the Web Monetization extension using real money, and you have a Fynbos US wallet connected to your extension, then you have two options:
+
+1. Visit and pay any of these monetized websites:
+
+   - [jeremiahLee.com](http://jeremiahLee.com)
+   - [lifebe.com.au](https://lifebe.com.au/)
+   - [storytogo.ca/classroom](https://storytogo.ca/classroom/)
+
+2. Visit the Web Monetization Playground, and add any Fynbos US wallet address. Here are examples that you can use:
```

Comment:
Same comment here re: being clear these are wallet addresses, not websites. Same comment applies to the rest of the sections, where appropriate.

Reply:
Awesome, addressed.